packages/salesforcedx-vscode-core/test/vscode-integration/commands/util/parameterGatherers.test.ts

```diff
@@ -25,7 +25,7 @@ import {
 import { SfdxPackageDirectories } from '../../../../src/sfdxProject';
 import { getRootWorkspacePath } from '../../../../src/util';
 
-const SFDX_SIMPLE_NUM_OF_DIRS = 14;
+const SFDX_SIMPLE_NUM_OF_DIRS = 16;
```

Comment:
Why did this change?

Reply:
There are 2 new dirs: `force-app/main/default/lwc/demoLwcComponent/`, `force-app/main/default/lwc/demoLwcComponent/__tests__`